stack.py

```diff
@@ -58,21 +58,26 @@ def parse_program(code):
         if not parts:
             continue
         op = parts[0]
-        args = [float(arg) for arg in parts[1:]]
+        args = [try_float(arg) for arg in parts[1:]]
         yield Instr(op, args)
 
 
 def eval_program(program):
     instuctions = parse_program(program)
     stack = []
+    
     for instr in instuctions:
         if instr.op == "push":
             stack.append(instr.args[0])
         elif instr.op == "pop":
             stack.pop()
         elif is_operator(instr.op):
-            b = stack.pop()
-            a = stack.pop()
+            if 'quiet' in instr.args:
```

Comment:
I disagree with your treatment of `quiet` as an argument, this is what I added the `prefix` list for.